sharness/t0030-ctl-pin.sh

```diff
@@ -49,6 +49,34 @@ test_expect_success IPFS,CLUSTER "wait for data to unpin from cluster with ctl w
     ipfs-cluster-ctl status "$cid" | grep -q -i "UNPINNED"
 '
 
+test_expect_success IPFS,CLUSTER "pin data to cluster with ctl using ipfs paths" '
+    cid=(`docker exec ipfs sh -c "mkdir -p test1/test2 && touch test1/test2/test3.txt && ipfs add -qr test1"`)
```

Comment:
Can this line be placed right before the test? It's weird that tests below use a variable set in a different test. I think it should work but not 100% sure.

Reply:
>  I think it should work but not 100% sure.

It works. In a test you can read variable read in tests below. In fact, the tests in a file share the same environment. Things pinned in one test are unpinned in another. Individual tests are not independent, but I think they make sense  that way. And anyway it's not like you can run those tests individually, but all tests in a file. But if that doesn't seem like a nice pattern let me know.

Reply:
I know it works like this, but removing/changing a single test will cause a cascade breakage in all the tests, so if this "intialization" line can be placed outside the tests at the top, better.